homework/inner-product/arithmeticAverage.hpp

```diff
@@ -0,0 +1,7 @@
+#pragma once
+#include <cmath>
+#include <vector>
+#include <numeric>
```

Comment:
numeric & cmath should be moved to cpp file :) They are not used in hpp

Reply:
OK :)